packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/handle-authentication-responses/react-native/handle-responses.md

```diff
@@ -0,0 +1,77 @@
+### Response handling
+
+The SDK provides two options, Promise and Event listener, to handle responses. You can choose either approach based on your preference.
+
+#### Handle response in Promise
+
+The `signIn` function will return a Promise if user credentials (username, password) have been provided. Success and error responses can be easily handled in standard Promise way.
+
+```javascript
+import { signIn } from '@okta/okta-react-native';
+
+class LoginScreen extends React.Component {
+
+    /* code from last section */
+
+    login() {
+        const { username, password } = this.state;
+        signIn({ username, password })
+            .then(token => {
+                // handle success response
+            })
+            .catch(error => {
+                // handle error response
+            });
+    }
+
+    /* code from last section */
+}
+```
+
+#### Handle response in Event listener
+
+You can also register event listeners from the `EventEmitter` function in the SDK to handle responses.
+
+**Note:** Please make sure listeners will be properly unregistered when the component will unmount.
```

Comment:
we avoid the use of "please" in content, and avoid future tense. Bolded the chagnes:

**Note**: Make sure **that** listeners **are** properly unregistered when the component **unmounts**.

Reply:
Thanks for the comment! I still want to keep the `will unmount` part due to https://github.com/okta/okta-developer-docs/pull/1093#discussion_r416918598